### ISSUE TITLE
Allow change of MQTT port

### ISSUE THREAD

**Issue by Orfait** [OpenEVSE/ESP8266_WiFi_v2.x#197]
Hi,

It would be very nice to be able to change the MQTT port, it is not always 1883.

May this feature be added in the future ?

Thank you
Albin

**Comment by glynhudson** [comment]
Yes, this is a good suggestion. We will look at adding this in the future. For the moment it can be changed by editing the following line then compiling and uploading:

https://github.com/OpenEVSE/ESP8266_WiFi_v2.x/blob/a5a43a9f025b00d85671e49b0e42f0fab19d173b/src/mqtt.cpp#L96
